android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -214,19 +210,90 @@ private static PackageInfo getPackageInfo(Context ctx) {
             logger.info("Error getting package info.");
             return null;
         }
+      }
+
+      logger.info(String.format("Native Integrations '%s'",
+                                options.getIntegrations().toString()));
+      sentryOptions = options;
+    });
+
+    promise.resolve(true);
+  }
+
+  @ReactMethod
+  public void setLogLevel(int level) {
+    logger.setLevel(this.logLevel(level));
+  }
+
+  @ReactMethod
+  public void crash() {
+    throw new RuntimeException(
+        "TEST - Sentry Client Crash (only works in release mode)");
+  }
+
+  @ReactMethod
+  public void fetchRelease(Promise promise) {
+    WritableMap release = Arguments.createMap();
+    release.putString("id", packageInfo.packageName);
+    release.putString("version", packageInfo.versionName);
+    release.putString("build", String.valueOf(packageInfo.versionCode));
```

Comment:
versionCode is deprecated depending OS version, see https://github.com/getsentry/sentry-java/blob/main/sentry-android-core/src/main/java/io/sentry/android/core/ContextUtils.java#L30-L47